src/main/dml/fenixedu-academic.dml

```diff
@@ -7139,7 +7139,9 @@ relation ExecutionCourseSummary {
 }
 
 relation ExecutionCourseSender {
-    ExecutionCourse playsRole course;
+    ExecutionCourse playsRole course {
+        multiplicity 1..1;
+    }
```

Comment:
This can't be done in a non major version since you cannot assume the current data is consistent.